props/saber_fett263_buttons.h

```diff
@@ -285,6 +445,215 @@ EFFECT(faston); // for EFFECT_FAST_ON
 EFFECT(blstbgn); // for Begin Multi-Blast
 EFFECT(blstend); // for End Multi-Blast
 EFFECT(push); // for Force Push gesture in Battle Mode
+EFFECT(mnum); // menu numbers
+EFFECT(quote); // quote on force effect
+#ifdef FETT263_EDIT_MODE_MENU
+EFFECT(medit); // Edit Mode
+#endif
+
+class GestureControlFile : public ConfigFile {
+public:
+  void SetVariable(const char* variable, float v) override {
+    CONFIG_VARIABLE(gestureon, 1);
+#ifdef SWING_GESTURE
+    CONFIG_VARIABLE(swingon, 1);
+#else
+    CONFIG_VARIABLE(swingon, 0);
+#endif
+    CONFIG_VARIABLE(swingonspeed, FETT263_SWING_ON_SPEED);
+#ifdef TWIST_GESTURE
+    CONFIG_VARIABLE(twiston, 1);
+#else
+    CONFIG_VARIABLE(twiston, 0);
+#endif
+#ifdef THRUST_GESTURE
+    CONFIG_VARIABLE(thruston, 1);
+#else
+    CONFIG_VARIABLE(thruston, 0);
+#endif
+#ifdef STAB_GESTURE
+    CONFIG_VARIABLE(stabon, 1);
+#else
+    CONFIG_VARIABLE(stabon, 0);
+#endif
+#ifdef FETT263_TWIST_OFF
+    CONFIG_VARIABLE(twistoff, 1);
+#else
+    CONFIG_VARIABLE(twistoff, 0);
+#endif
+#ifdef FETT263_POWER_LOCK
+    CONFIG_VARIABLE(powerlock, 1);
+#else
+    CONFIG_VARIABLE(powerlock, 0);
+#endif
+#ifdef FORCE_PUSH_CONDITION
+    CONFIG_VARIABLE(forcepush, 1);
+#else
+    CONFIG_VARIABLE(forcepush, 0);
+#endif
+    CONFIG_VARIABLE(forcepushlen, FETT263_FORCE_PUSH_LENGTH);
+    CONFIG_VARIABLE(lockupdelay, FETT263_LOCKUP_DELAY);
+    CONFIG_VARIABLE(clashdetect, FETT263_BM_CLASH_DETECT);
+#ifdef FETT263_MAX_CLASH
+    CONFIG_VARIABLE(maxclash, FETT263_MAX_CLASH);
+#else
+    CONFIG_VARIABLE(maxclash, 16);
+#endif
+}
+  bool gestureon; // gesture controls on/off for use with "Gesture Sleep"
+  bool swingon; // Swing On ignition
+  int swingonspeed; // Swing On speed (200 ~ 500 range)
+  bool twiston; // Twist On ignition
+  bool thruston; // Thrust On ignition
+  bool stabon; // Stab On ignition
+  bool forcepush; // Force Push
+  int forcepushlen; // Force Push Length
+  int lockupdelay; // Lockup Delay (for Battle Mode)
+  bool twistoff; // Twist Off retraction
+  bool powerlock; // diable PWR button for retraction, for use with "Power Lock" mode to prevent button turning saber off
+  int clashdetect; // maximum Clash Strength to detect Clash during Battle Mode (0 ~ 10 range)
+  int maxclash; // maximum Clash Strength for Clash Sound and Detection works with CLASH_THRESHOLD_G to create range of Clash Strength (8 ~ 16 range)
+};
+
+#ifdef FETT263_SAVE_CHOREOGRAPHY
+// Rehearsal / Choreography
+class SavedRehearsal : public ConfigFile {
+public:
+  virtual ReadStatus Read(FileReader* f) override {
+    int file;
+    if (!f || !f->IsOpen()) return ReadStatus::READ_FAIL;
+    memset(clash_rec, 0, sizeof(clash_rec));
+    STDOUT.println("Reading choreo.ini");
+    for (size_t i = 0; i < NELEM(clash_rec); i++) {
+      char action[33];
+      f->skipwhite();
+      f->readVariable(action);
+      if (!strcmp(action, "clash")) {
+          if (SFX_clsh) {
```

Comment:
4-step indentation -> 2-step indentation (unless it's just github being weird.)

Reply:
fixed